extension/content.js

```diff
@@ -282,6 +284,78 @@ $(document).on('click', event => {
 	$target.closest('.js-details-container').toggleClass('refined-github-minimized');
 });
 
+const diffFile = (() => {
+	let lastFile;
+
+	const hasChanged = nextFile => {
+		if (nextFile !== lastFile) {
+			lastFile = nextFile;
+			return true;
+		}
+
+		return false;
+	};
+
+	const reset = function reset() {
+		lastFile = '';
+	};
+
+	return {
+		hasChanged,
+		reset
+	};
+})();
+
+function diffHeaderFilename() {
+	const targetDiffFile = getHighestVisibleDiffFilename();
+	if (!targetDiffFile) {
+		return;
+	}
+
+	const filename = $(targetDiffFile).find('.file-header').attr('data-path');
+
+	if (!diffFile.hasChanged(filename)) {
+		return;
+	}
+
+	$('.diff-toolbar-filename').text(`Filename: ${filename}`);
```

Comment:
Is there really any need to prefix it with `Filename: `? That's pretty obvious to me at least.